tests/unit/test_display.py

```
@@ -0,0 +1,347 @@
+
+Tests terminal UI formatting and display.
+"""
+
+import unittest
+from datetime import datetime, timedelta
+from unittest.mock import Mock, patch, MagicMock
+from io import StringIO
+
+from gh_pr.ui.display import DisplayManager as PRDisplay
+# Mock other classes that don't exist
+class TableDisplay: pass
```

Ruff (0.13.1)

Multiple statements on one line (colon)

(E701)